example-scripts/voctolight/lib/plugins/stdout.py

```diff
@@ -0,0 +1,13 @@
+# Plugin to provide a tally light interface via stdout
+__all__ = ['Stdout']
+
+class Stdout:
+    def __init__(self, config):
+        # We don't need an explicit event loop pump
+        self.pump = lambda: None
```

Comment:
Ok, now I'm certainly confused. When neither the RPi nor the Stdout-Plugin ned this facility, why is it there in the first place? Also the Plugin should not need to worry about facilities it does not use (see RPi plugin)

Reply:
This was used by a GTK version, not included in these PRs.  I will remove this.